qgreenland/config/helpers/layers/continental_shelf.py

```diff
@@ -0,0 +1,105 @@
+from qgreenland.models.config.layer import Layer, LayerInput
+from qgreenland.models.config.step import CommandStep
+
+
+LAYER_PARAMS = {
+    'north_points': {
+        'title': 'North (points)',
+        'description': (
+            """Points representing the boundary of the Northern extended continental shelf
+            of Greenland."""
+        ),
+    },
+    'north_lines': {
+        'title': 'North (lines)',
+        'description': (
+            """Lines representing the boundary of the Northern extended continental shelf of
+            Greenland."""
+        ),
+    },
+    'north_polygons': {
+        'title': 'North (poloygons)',
+        'description': (
+            """Polygons representing the area between the Greenland Exclusive
+            Economic Zone and the Northern Continental Shelf."""
+        ),
+    },
+    'northeast_points': {
+        'title': 'Northeast (points)',
+        'description': (
+            """Points representing the boundary of the Northeastern extended continental shelf
+            of Greenland."""
+        ),
+    },
+    'northeast_lines': {
+        'title': 'Northeast (lines)',
+        'description': (
+            """Lines representing the boundary of the Northeastern extended continental shelf
+            of Greenland."""
+        ),
+    },
+    'northeast_polygons': {
+        'title': 'Northeast (polygons)',
+        'description': (
+            """Polygons representing the area between the Greenland Exclusive
+            Economic Zone and the Northeastern Continental Shelf."""
+        ),
+    },
+    'south_points': {
+        'title': 'South (points)',
+        'description': (
+            """Points representing the boundary of the Southern extended continental shelf
+            of Greenland."""
+        ),
+    },
+    'south_lines': {
+        'title': 'South (lines)',
+        'description': (
+            """Lines representing the boundary of the Southern extended continental shelf of
+            Greenland."""
+        ),
+    },
+    'south_polygons': {
+        'title': 'South (polygons)',
+        'description': (
+            """Polygons representing the area between the Greenland Exclusive
+            Economic Zone and the Southern Continental Shelf."""
+        ),
+    },
+}
+
+
+def make_layers() -> list[Layer]:
+    return [
+        Layer(
+            id=f'continental_shelf_{key}',
+            title=params['title'],
+            description=params['description'],
+            tags=[],
+            input=LayerInput(
+                dataset=dataset,
+                asset=dataset.assets[key],
+            ),
+            steps=[
+                CommandStep(
+                    args=[
+                        'unzip',
+                        '{input_dir}/*.zip',
+                        '-d',
+                        '{output_dir}',
+                    ],
+                ),
+                CommandStep(
```

Comment:
Looks like we can use the `ogr2ogr` helper here, `qgreenland.config.helpers.steps.ogr2ogr`.

Reply:
Nevermind, the default `ogr2ogr` helper requires a clip boundary. We don't want to clip this layer.